makefiles/pseudomodules.inc.mk

```diff
@@ -207,6 +207,49 @@ PSEUDOMODULES += senml_phydat
 PSEUDOMODULES += senml_saul
 PSEUDOMODULES += sha1sum
 PSEUDOMODULES += sha256sum
```

Comment:
Those would need to be converted to the new naming scheme too

Reply:
Done